modules/seqexec/model/src/test/scala/seqexec/model/ModelLensesSpec.scala

```diff
@@ -64,4 +64,12 @@ final class ModelLensesSpec extends CatsSuite with ModelLenses {
            OptionalTests(instrumentDisperserLambdaO))
   checkAll("instrument observing mode Optional",
            OptionalTests(instrumentObservingModeO))
+  checkAll("instrument disperser mode lambda Optional",
+           OptionalTests(instrumentDisperserLambdaO))
+  checkAll("instrument decker Optional",
+           OptionalTests(instrumentDeckerO))
```

Comment:
I guess decker and disperser mode were omitted previously?

Reply:
Yes, I forgot to add tests to those lenses